contents/barnsley/code/rust/Cargo.toml

```
@@ -1,9 +1,13 @@
 [package]
-name = "rust"
+name = "barnsley"
 version = "0.1.0"
 edition = "2018"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-rand = "0.8.4"
\ No newline at end of file
+rand = "0.8.4"
+
+[[bin]]
+path = "./barnsley.rs"
+name = "main"
```

I would use the file name to make things clearer.

```suggestion
name = "barnsley"
```